patitasuy/src/app/pages/chat/chat.component.ts

```diff
@@ -1,14 +1,27 @@
 import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { Chat } from 'src/app/interfaces/chat';
+import { MessageService } from 'src/app/services/message.service';
 
 @Component({
   selector: 'app-chat',
   templateUrl: './chat.component.html',
   styleUrls: ['./chat.component.scss'],
 })
-export class ChatComponent  implements OnInit {
+export class ChatComponent implements OnInit {
 
-  constructor() { }
+  messageId!: number;
+  constructor(private route: ActivatedRoute, private msgService: MessageService) { }
 
-  ngOnInit() {}
+  chat: Chat | undefined;
 
+
+  ngOnInit() {
+    this.route.params.subscribe(params => {
+      this.messageId = params['id'];
+      this.msgService.getChatById(Number(this.messageId)).subscribe(values => {
```

Comment:
this is ok but can be replace with "+this.nessageId"

Reply:
That's a bit obscure